test/MaeParserTest.cpp

```diff
@@ -123,23 +123,23 @@ BOOST_AUTO_TEST_CASE(BlockBeginning)
 {
     {
         auto ss = std::make_shared<std::stringstream>("m_something {");
-        int indexed = 0;
```

Comment:
why are these updated?

Reply:
I wanted to avoid any possible ambiguity if, for whatever reason, `indexed` came back with value 0. If it ever does (after another patch, I mean), it must have been set by `blockBeginning()`.

Reply:
oh, right. that makes sense. This makes the test more sensitive if we mess up. great!